sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1092,48 +1040,138 @@ protected override async Task<EventProcessorCheckpoint> GetCheckpointAsync(strin
         }
 
         /// <summary>
-        ///   Creates a <see cref="StorageManager" /> to use for interacting with durable storage.
-        /// </summary>
-        ///
-        /// <param name="checkpointStore">The client responsible for interaction with durable storage, responsible for persisting checkpoints and load-balancing state.</param>
-        ///
-        /// <returns>A <see cref="StorageManager" /> with the requested configuration.</returns>
-        ///
-        private StorageManager CreateStorageManager(BlobContainerClient checkpointStore) => new BlobsCheckpointStore(checkpointStore, RetryPolicy);
-
-        /// <summary>
-        ///   Processes the starting positions for checkpoints, ensuring that any overrides set by the <see cref="PartitionInitializingAsync" />
-        ///   handler are respected when no natural checkpoint exists for the partition.
+        ///   Signals the <see cref="EventProcessorClient" /> to begin processing events.  Should this method be called while the processor
+        ///   is running, no action is taken.
```

Comment:
```suggestion
        ///   Signals the <see cref="EventProcessorClient" /> to begin processing events.  If the processor is already
        ///   running, this method has no effect.
```

Reply:
Wow... that was some awkward phrasing and has been here for a while.